core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/builtin_argument/validators.go

```diff
@@ -110,3 +110,36 @@ func DurationOrNone(value starlark.Value, attributeName string) *startosis_error
 	}
 	return nil
 }
+
+func RelativeOrRemoteAbsoluteLocator(locatorStarlarkValue starlark.Value, packageId string, argNameForLogging string) *startosis_errors.InterpretationError {
+
+	locatorStarlarkStr, ok := locatorStarlarkValue.(starlark.String)
+	if !ok {
+		return startosis_errors.NewInterpretationError("Value for '%s' was expected to be a starlark.String but was '%s'", argNameForLogging, reflect.TypeOf(locatorStarlarkValue))
+	}
+	locatorStr := locatorStarlarkStr.GoString()
+
+	if isRelativeLocator(locatorStr) {
```

Comment:
Why do you need to check first if it's relative? Checking only for `isLocalLocator` should be enough no?

Reply:
yeap! you are right! we can just check if it's a local-absolute-locator or not